db/keyval/redis/plugin_impl.go

```diff
@@ -56,23 +92,3 @@ func (p *Plugin) retrieveConfig() (cfg interface{}, err error) {
 	}
 	return cfg, nil
 }
-
```

Comment:
Let's not change the order of methods because then it is hard to review & compare.